charts/cass-operator/templates/configmap.yaml

```diff
@@ -35,15 +35,20 @@ data:
     images:
       system-logger: "k8ssandra/system-logger:v1.10.0"
       config-builder: "datastax/cass-config-builder:1.0.4-ubi7"
+{{- if .Values.image.repositoryOverride }}
+    {{- with .Values.image.repositoryOverride }}
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+{{- end }}
 {{- if .Values.image.registryOverride }}
-    {{ indent 2 (cat "imageRegistry:" .Values.image.registryOverride) }}
+    imageRegistry: {{ .Values.image.registryOverride }}
 {{- end }}
-{{- with .Values.imagePullSecrets }}
-    imagePullSecrets:
+{{- with .Values.imagePullSecret }}
```

Comment:
Since `imagePullSecrect` is a just a scalar value, I would rewrite this `with` block to be just like the `if` block for `imageRegistry`.